### PR TITLE
Better user handling

### PR DESCRIPTION
**Pull request by xouillet**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/blatinier/RendezMoiMesPlumes/pull/118%23discussion_r140661577%22%2C%20%22https%3A//github.com/blatinier/RendezMoiMesPlumes/pull/118%23discussion_r140665071%22%2C%20%22https%3A//github.com/blatinier/RendezMoiMesPlumes/pull/118%23issuecomment-335039289%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/blatinier/RendezMoiMesPlumes/pull/118%23issuecomment-335039289%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Merged%21%22%2C%20%22created_at%22%3A%20%222017-10-08T21%3A18%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/1086629%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/blatinier%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2095b2d284c7368f8465f2a58551290bf8eeb5d025%20pigeon/weights/templates/registration/register.html%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/blatinier/RendezMoiMesPlumes/pull/118%23discussion_r140661577%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22No%20way%20to%20just%20do%20%60in%20%5B%27language%27%2C%20%27country%27%2C%20%27avatar%27%5D%60%3F%22%2C%20%22created_at%22%3A%20%222017-09-24T16%3A57%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/1086629%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/blatinier%22%7D%7D%2C%20%7B%22body%22%3A%20%22Nope%2C%20thanks%20to%20the%20wonderful%20template%20language%20of%20django.%20This%20solution%20is%20a%20well%20known%20quick%20and%20dirty%20workaround.%20A%20cleaner%20solution%20could%20be%20to%20define%20the%20list%20in%20the%20view%20and%20use%20the%20variable%20in%20template%20but%20it%20is%20less%20understandable%20imo.%22%2C%20%22created_at%22%3A%20%222017-09-24T19%3A20%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/310587%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xouillet%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20pigeon/weights/templates/registration/register.html%3AL6-18%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 95b2d284c7368f8465f2a58551290bf8eeb5d025 pigeon/weights/templates/registration/register.html 10'></a> &nbsp;<b><a href='https://github.com/blatinier/RendezMoiMesPlumes/pull/118#discussion_r140661577'>File: pigeon/weights/templates/registration/register.html:L6-18</a></b>
- <a href='https://github.com/blatinier'></a> No way to just do `in ['language', 'country', 'avatar']`?
- <a href='https://github.com/xouillet'></a> Nope, thanks to the wonderful template language of django. This solution is a well known quick and dirty workaround. A cleaner solution could be to define the list in the view and use the variable in template but it is less understandable imo.
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/blatinier/RendezMoiMesPlumes/pull/118#issuecomment-335039289'>General Comment</a></b>
- <a href='https://github.com/blatinier'></a> Merged!

<a href='https://www.codereviewhub.com/blatinier/RendezMoiMesPlumes/pull/118?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/blatinier/RendezMoiMesPlumes/pull/118?approve=1'></a>&nbsp;<a href='https://github.com/blatinier/RendezMoiMesPlumes/pull/118'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by blatinier** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/blatinier/RendezMoiMesPlumes/pull/118'>Refresh</a>.<a href='#crh-update-none'></a>

File: pigeon/weights/templates/registration/register.html [hunk 1]
```diff
@@ -6,11 +6,12 @@
   <h1 class="text-center col-lg-12 pigeon-font pigeon-color margin-bottom-20">{% trans "Register" %}</h1>
   <form method="post" autocomplete="off" class="form-contact col-lg-offset-2">
     {% csrf_token %}
-    {% for field in user_form %}
-      {% bootstrap_field field layout="horizontal" form_group_class="col-md-12" field_class="col-md-9 input-form-space" show_help=False show_label=False %}
-    {% endfor %}
-    {% for field in profile_form %}
-      {% bootstrap_field field layout="horizontal" form_group_class="col-md-12" field_class="col-md-8 input-form-space" label_class="register-label col-md-1" show_help=False %}
+    {% for field in register_form %}
+      {% if field.name in 'language,country,avatar'|slice:',' %}
```
Comment (blatinier) [inline pigeon/weights/templates/registration/register.html:10]: No way to just do `in ['language', 'country', 'avatar']`?

Comment (xouillet) [reply pigeon/weights/templates/registration/register.html:10]: Nope, thanks to the wonderful template language of django. This solution is a well known quick and dirty workaround. A cleaner solution could be to define the list in the view and use the variable in template but it is less understandable imo.

**Comment by blatinier** [comment]
Merged!

**Pull request closed without merging**